e2e/README.md

```diff
@@ -43,4 +66,14 @@ Now it is possible to query the data with a known expected result thanks to the
 
 ```
 ./kishell search --newer="8760h" --query="clientip:172.155.107.128"
-```
\ No newline at end of file
+```
+
+## Troubleshoot
+
+If you have this error when you try to run```docker-compose up --build``` :
```

Comment:
I think we can omit the command in this line and just mention in plain English, like so:

"If you have this error when you try to spin up the stack:"

Reply:
Thanks for this @robsondepaula.